frontend/.gitignore

```diff
@@ -15,4 +15,5 @@ build-storybook.log
 maintenance.json
 diff.txt
 
-wiremock-jre8-standalone-2.29.1.jar
\ No newline at end of file
+wiremock-jre8-standalone-2.29.1.jar
```

Comment:
Would it be alright if we changed this to be more generic in case our versions change `wiremock*` and add it to the dockerignore? It's a sizeable file and will increase the time to setup our build context.